@here/harp-mapview/test/PoiRendererTest.ts

```diff
@@ -97,4 +103,88 @@ describe("PoiRenderer", function () {
             expect(screenBox.h).to.equal(16);
         });
     });
+
+    describe("search for cached images", function () {
+        const webGLRenderer = {
+            capabilities: {
+                isWebGL2: false
+            }
+        } as THREE.WebGLRenderer;
+        const mapViewStub = sinon.createStubInstance(MapView);
+        const poiManager = new PoiManager((mapViewStub as any) as MapView);
+        const testImageName = "testImage";
+        const mapEnv = new Env();
+
+        const createPointLabel = (name: string) => {
+            return {
+                poiInfo: {
+                    imageTextureName: name,
+                    isValid: true,
+                    buffer: undefined,
+                    technique: {}
+                },
+                visible: true
+            } as TextElement;
+        };
+
+        let x = 1;
+        let y = 1;
+
+        const addRandomImageToCache = (
+            testCache: MapViewImageCache,
+            name: string,
+            load: boolean
+        ) => {
+            // Note, the images must be unique, otherwise the test will fail, because the internal
+            // caching mechanism of the ImageCache will have already loaded the image.
+            return testCache.addImage(name, `https://picsum.photos/${x++}/${y++}`, load);
```

Comment:
Unit tests must not have external dependencies. We can instead stub testCache's findImageByName method and make it return what we want whenever we want in the test.

Reply:
Yes, very good point, I will fix.

Reply:
Done, I could use a local resource so I don't need to stub.